content/blog/adopting-existing-cloud-resources-into-pulumi/index.md

```diff
@@ -5,6 +5,7 @@ meta_desc: "Migrating from Terraform or another IaC tool? Learn how to adopt exi
 meta_image: "meta.png"
 authors: ["luke-hoban"]
 tags: ["migration"]
+canonical_url: "https://www.pulumi.com/docs/iac/concepts/options/import/"
```

Comment:
This one actually touches on multiple topics in multiple pages. The closest is probably this, but it's not the entirety of the content. (The content is outdated, though.):

```suggestion
canonical_url: "https://www.pulumi.com/docs/iac/adopting-pulumi/migrating-to-pulumi/from-terraform/"
```